packages/patternlab/styleguide/source/assets/js/modules/responsiveTable.js

```diff
@@ -109,32 +107,15 @@ export default (function (window, document, $) {
     if (rt.horizontalScrollable) {
       $tableWrapper.attr("tabindex", "0");
       $tableWrapper.attr("aria-labelledby", "sr-instructions");
-      $scrollInfo.addClass("show");
+      $stickyHeader.removeAttr("aria-labelledby");
```

Comment:
```suggestion
      $stickyHeader.removeAttr("aria-labelledby");
```

These should apply everywhere not just when the table has horizontal scrolling - please see my latest comment in the ticket. We no longer need `sr-instructions` and this block of conditional code  and these new attributes should be added directly in the twig template

Reply:
@arthurbaghdas I think these should not be wrapped in conditionals, instead they should be directly set in the html `"<div class='sticky-thead'><div class='sticky-thead-wrapper'><table></table></div></div>"` on line 49. 
For example, the second table (with sticky header but without horizontal scrolling) is not getting the aria-hidden and tabindex=-1. These should apply to all sticky table headers as they initiated